addons/share-through-mystuff/scratchr2.css

```diff
@@ -0,0 +1,3 @@
+.media-item-content.not-shared .media-action {
+  width: 75px;
+}
```

Comment:
Somewhat nitpicky, but I think the general rule of thumb for addon elements affected by other addons (like dark mode and 2.0->3.0) is to have the code that affects the addon elements in the addon that affects them, not in the addon that adds the element.

Reply:
![STMS-Scratchr2css-moved-confirmation](https://user-images.githubusercontent.com/75220768/221066821-b81fa54b-6771-4150-8519-41426bd2f927.png)
That?

Reply:
So rename `scratchr2.css`(probably to something like `share-through-mystuff.css`), move it to `/addons/scratchr2/`(and also edit the `addon.json`s to reflect these changes)?

Reply:
> So rename `scratchr2.css`(probably to something like `share-through-mystuff.css`), move it to `/addons/scratchr2/`(and also edit the `addon.json`s to reflect these changes)?

I mean merging scratchr2.css into `addons/scratchr2/mystuff.css` (without any `if`s).
`

Reply:
I don't think there's consensus about this specific case. There's #275 but specifically about dark mode.

Reply:
Things like this can usually be done without any `if`s. I made a PR that rewrites this addon's CSS: #5816